package/yast2-bootloader.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Mon Jun 29 12:46:58 UTC 2015 - jreidinger@suse.com
+
+- set only proper boot boot flags, otherwise it can confuse some 
```

Comment:
The term `proper boot flags` is a bit vague, could you be more specific if possible?